cog/tests/test_daily_charge.py

```
@@ -0,0 +1,161 @@
+            "2025-03-14 00:00:00",
+            "2025-03-14 23:59:59",
+        ]
+
+        test_cases_false = [
+            "2025-03-12 23:59:59",
+            "2025-03-15 00:00:00",
+        ]
+
+        for last_charge in test_cases_true:
+            with self.subTest(last_charge=last_charge):
+                self.assertTrue(
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)